core/CMakeLists.txt

```diff
@@ -0,0 +1,144 @@
+# Build libdronecore
+
+# This finds thread libs on Linux, Mac, and Windows.
+find_package(Threads REQUIRED)
+
+include_directories(
+    SYSTEM ${CMAKE_SOURCE_DIR}/libs/include
+)
+
+if (ANDROID)
+    include_directories(curl-android-ios/prebuilt-with-ssl/android/include)
+endif()
+
+if (IOS)
+    include_directories(curl-android-ios/prebuilt-with-ssl/ios/include)
+endif()
+
+if (IOS OR ANDROID OR MSVC OR APPLE)
+    set(library_type "STATIC")
+
+    # We need tinyxml2 for the camera definition parsing.
+    # We use the submodule when linking statically.
+    add_subdirectory(libs/tinyxml2 EXCLUDE_FROM_ALL)
+    include_directories(SYSTEM libs/tinyxml2)
+else()
+    # Use tinyxml2 from the host system for Linux.
```

Comment:
@julianoes This comment does not apply to the next line, which says that libdronecore will be a shared library, right?

Reply:
That's true. What's your suggestion?

Reply:
Removing the comment xD.